pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -160,6 +184,37 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	return nil
 }
 
+func unique(logger *logrus.Entry, a, b []string) []string {
```

Comment:
Would be nice if we can have a few lines of comments above to describe what this func does.